packages/cfd/src/Containers/__tests__/cfd-top-up-demo-modal.spec.js

```diff
@@ -0,0 +1,163 @@
+        account_type: 'financial',
+        leverage: 1000,
+        short_title: 'Financial',
+    };
+
+    const financial_stp_config = {
+        account_type: 'financial_stp',
+        leverage: 100,
+        short_title: 'Financial STP',
+    };
+
+    const mock_props = {
+        dxtrade_companies: {},
+        mt5_companies: {
+            demo: {
+                synthetic: {
+                    mt5_account_type: synthetic_config.account_type,
+                    leverage: synthetic_config.leverage,
+                    title: 'Demo Synthetic',
+                    short_title: synthetic_config.short_title,
+                },
+                financial: {
+                    mt5_account_type: financial_config.account_type,
+                    leverage: financial_config.leverage,
+                    title: 'Demo Financial',
+                    short_title: financial_config.short_title,
+                },
+                financial_stp: {
+                    mt5_account_type: financial_stp_config.account_type,
+                    leverage: financial_stp_config.leverage,
+                    title: 'Demo Financial STP',
+                    short_title: financial_stp_config.short_title,
+                },
+            },
+            real: {
+                synthetic: {
+                    mt5_account_type: synthetic_config.account_type,
+                    leverage: synthetic_config.leverage,
+                    title: 'Synthetic',
+                    short_title: synthetic_config.short_title,
+                },
+                financial: {
+                    mt5_account_type: financial_config.account_type,
+                    leverage: financial_config.leverage,
+                    title: 'CFDs',
+                    short_title: financial_config.short_title,
+                },
+                financial_stp: {
+                    mt5_account_type: financial_stp_config.account_type,
+                    leverage: financial_stp_config.leverage,
+                    title: 'Financial STP',
+                    short_title: financial_stp_config.short_title,
+                },
+            },
+        },
+        current_account: { category: 'demo', type: 'financial', balance: '700', display_balance: '700' },
+        closeSuccessTopUpModal: jest.fn(),
+        closeTopUpModal: jest.fn(),
+        is_top_up_virtual_open: true,
+        is_top_up_virtual_in_progress: false,
+        is_top_up_virtual_success: false,
+        platform: 'test platform',
+        topUpVirtual: jest.fn(),
+    };
+
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    it('should render <CFDTopUpDemoModal /> correctly', () => {
+        render(<CFDTopUpDemoModal {...mock_props} />);
```

Comment:
There is no assertions here. So the this test will pass even if the component is not rendered.